<launch>
  <include file="$(find realsense2_camera)/launch/rs_camera.launch"/>
  <group ns="camera/color">
    <node name="marker_pose" pkg="softgrasp_ros" type="marker_pose" required="true" output="screen">
      <remap from="image" to="image_rect_color"/>
    </node>
    <node name="image_proc" pkg="image_proc" type="image_proc" required="true" output="screen"/>
    <node name="image_view" pkg="image_view" type="image_view" required="true" output="screen">
      <remap from="image" to="image_marker_pose"/>
    </node>
  </group>
</launch>